lib/remote_retro_web/templates/layout/app.html.eex

```diff
@@ -70,13 +70,17 @@
             <p>RemoteRetro</p>
           </div>
         </a>
-        <div class="ui right floated header right-branded-header">
+        <div class="ui right floated header right-branded-header grid">
           <a href="http://www.stridenyc.com/" target="_blank">
             <div>
               <div class="middle aligned content logo-text">Built by</div>
               <div class="ui tiny image"><img src="<%= static_path(@conn, "/images/stride_logo.png") %>" alt="Stride" /></div>
             </div>
           </a>
+
+          <a href="/logout" class="ui button mini">
```

Comment:
@kamok Realizing that we'll only want to display this button if there _is_ a current user, as it'll show up in the `/faq` page

Reply:
lemmeknow if you want to take that on, or if you want me to add it this weekend? happy to do it, as it was an oversight on my part

Reply:
I can do it. Should be straightforward.

Reply:
Doing it.